instance.tf

```diff
@@ -1,6 +1,6 @@
 resource "google_sql_database_instance" "instance" {
   database_version     = var.database_version
-  name                 = "${var.project_prefix}-${var.instance_name}"
+  name                 = "${var.project_prefix != null ? "${var.project_prefix}-" : ""}${var.instance_name}"
```

Comment:
Having the conditional stuff within the actual string looks a bit weird, especially with the nested quoting. A cleaner approach, to me at least, would be:

```terraform
  name                 = ${var.project_prefix != null ? "${var.project_prefix}-${var.instance_name}" : "${var.instance_name}"
```

Which makes me wonder what would be considered a better practice, the above or:
```terraform
  name                 = ${var.project_prefix == null ? "${var.instance_name}" : "${var.project_prefix}-${var.instance_name}"
```

:thinking: 

I personally have a slight preference for the latter.